### ISSUE TITLE
[ObjectStatus]: A11y-ObjectStatus fails to meet 4:5:1 contrast ratio

### ISSUE THREAD

**Issue by tellenberger** [SAP/ui5-webcomponents-react#7391]
### Describe the bug

This issue is relevant to EAA compliance starting June 28, 2025
Related to this bug: https://github.com/SAP/ui5-webcomponents-react/issues/5079 but limited to error objectstatus but also success and warning.

### Reproduction steps

1. examine the foreground color for any ValueState ObjectStatus object
2. examine the background color for any ValueState ObjectStatus object
3. compute the color contrast ratio

### Expected Behaviour

Expected contrast ratio should be at least 4:5:1.

### Screenshots or Videos

![Image](https://github.com/user-attachments/assets/b20d785f-52cd-46ed-b2fa-fc3f3bc2700c)

### UI5 Web Components for React Version

2.10.0

### UI5 Web Components Version

2.10.0

### Browser

Chrome

### Operating System

any

### Additional Context

Came up as an accessibility bug from internal testing for EAA compliance:

https://jira.tools.sap/browse/OMS-16419

### Relevant log output

```shell

```

### Organization

SAP

### Declaration

- [x] I’m not disclosing any internal or sensitive information.
